main.rb

```diff
@@ -0,0 +1,27 @@
+# require_relative 'app'
+
+# def main
+#   app = App.new
+#   interface
+#   selection = gets.chomp
+#   case selection
+#   when '1' then app.hello
+#   else
+#     puts 'Invalid selection'
+#     interface
+#   end
+# end
+
+# def interface
+#   puts "Welcome to School Library App!\n\n"
+#   puts 'Please choose an option by entering a number:'
+#   puts '1 - List all books'
+#   puts '2 - List all people'
+#   puts '3 - Create a person'
+#   puts '4 - Create a book'
+#   puts '5 - Create a rental'
+#   puts '6 - List all rentals for a given person id'
+#   puts '7 - Exit'
+# end
+
+# main
```

Comment:
- [ ] Your app is not running because you have commented the code on the `main.rb` file that is the entry point of your app, Please make sure to uncomment the code from `main.rb` file so users can run the app and select options from the interface.